docs/.gdnbaselines

```diff
@@ -11,7 +11,9 @@
     "552baaa9d24596ed82e97b538dbef8abfcc9a0a8a9fdc8d3423424bf003b4fa7": {
       "signature": "552baaa9d24596ed82e97b538dbef8abfcc9a0a8a9fdc8d3423424bf003b4fa7",
       "alternativeSignatures": [
-        "b5f214e29b5462952a2841914ef8d1fc8141a9e8a69ea2b2be6c0cd409bf15d7"
+        "b5f214e29b5462952a2841914ef8d1fc8141a9e8a69ea2b2be6c0cd409bf15d7",
+        "f7352489c50aed5529c30f15f2d6e019dc840612ceb612ea5ffb44a0d994c641",
+        "956ef6761cd88e6cc39f6bd892c0e1e0719ac955d3e4bd5a52189ab9e64db97f"
```

Comment:
New versions of errors that were baselined earlier. Dependencies were updated in another commit which changed the signatures of the errors. Long term we need to run these validations on PR so failures don't creep in. But that can wait. For now deployments just need to fail if the guardian tasks fail.